app/src/main/java/com/zulip/android/util/PrefManager.java

```diff
@@ -0,0 +1,33 @@
+package com.zulip.android.util;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+
+
+public class PrefManager {
+
+    // Shared preferences file name
+    private static final String PREF_NAME = "zulip-slides";
+    private static final String IS_FIRST_TIME_LAUNCH = "IsFirstTimeLaunch";
+    SharedPreferences pref;
+    SharedPreferences.Editor editor;
+    Context _context;
+    // shared pref mode
+    int PRIVATE_MODE = 0;
+
+    public PrefManager(Context context) {
+        this._context = context;
+        pref = _context.getSharedPreferences(PREF_NAME, PRIVATE_MODE);
+        editor = pref.edit();
+    }
+
+    public boolean isFirstTimeLaunch() {
+        return pref.getBoolean(IS_FIRST_TIME_LAUNCH, true);
+    }
+
+    public void setFirstTimeLaunch(boolean isFirstTime) {
+        editor.putBoolean(IS_FIRST_TIME_LAUNCH, isFirstTime);
```

Comment:
how about saving the version of app instead of a boolean; shared pref will remain same across upgrades